simconvoi.cc

```diff
@@ -5373,3 +5395,24 @@ void convoi_t::next_stop_button_pressed() {
 	}
 }
 
+void convoi_t::set_convoi_coupling_in_progress(convoihandle_t convoi_coupling_undergo) {
+	if( !convoi_coupling_undergo.is_bound() ) {
+		dbg->warning( "convoi_t::set_convoi_coupling_in_progress()","%i cannot find the coupling convoi!", self.get_id());
+		return;
+	} else {
+		convoi_coupling_in_progress=convoi_coupling_undergo;
+		dbg->message( "convoi_t::set_convoi_coupling_in_progress()","%i and %i convoys will be coupling soon", self.get_id(), convoi_coupling_in_progress->self.get_id() );
+		return;
+	}
```

Comment:
elseブロックは不要です。（early returnの目的はインデントを浅くすることです）
```suggestion
	}
	convoi_coupling_in_progress=convoi_coupling_undergo;
	dbg->message( "convoi_t::set_convoi_coupling_in_progress()","%i and %i convoys will be coupling soon", self.get_id(), convoi_coupling_in_progress->self.get_id() );
```